tools/llvm-blocks-dump.cpp

```diff
@@ -0,0 +1,60 @@
+#include "dg/analysis/PointsTo/PointerAnalysisFI.h"
```

Comment:
A better name would be `llvm-ntscd-dump` or something like that. `llvm-blocks-dump` really does not say what it does.